### PR TITLE
Add 1280 VAS for 5000

### PR DESCRIPTION
**Pull request by pedromacli**
1280 for 5000

### CODE REVIEW

**Comment by bringdev** [comment]
[trello.com/c/zUfiCynk](https://trello.com/c/zUfiCynk)

File: data/services_vas.json [hunk 1]
```diff
@@ -1658,6 +1658,16 @@
     "domesticAllowedIn" : "NO",
     "senderCountries" : "-",
     "destinations" : "-"
+  }, {
+    "serviceName" : "Pakke til bedrift",
```
Comment (tomgag) [inline data/services_vas.json:1662]: Isn't this file auto-generated? If so, it will be overwritten next time anyone generates is from the common lib.

**Review by tomgag — APPROVED** [review]
LGTM

File: data/services_vas.json (hunk 1 above)
Comment (pedromacli) [reply data/services_vas.json:1662]: Yep, generated from the common lib 😉

**Pull request merged**